### DISCUSSION TITLE
Installation Error

### DISCUSSION THREAD

**Discussion by Borsoi** [jhassan8/crunchyroll-tizen#43] [Q&A]
I had an error during the execution of the installation of the online version of the application, an error occurred during the installation of the same, as in the attachment below, is there any place that I can capture the complete log and share with you about what happened?

![Error tizen instalation](https://github.com/jhassan8/crunchyroll-tizen/assets/63875868/2558a25a-3e91-43c8-8eb1-beba35626b6e)

**Comment by TONYSOAD05** [comment]
That error looks like that is because Tizen Studio was not installed in the Default Folder

**Reply by Borsoi** [reply]
I was searching about this error (118) it seems it is a certification error, I'll try to create a Tizen certificate an try to install it again

**Reply by jhassan8** [reply]
Hi @Borsoi, sorry for the delay, I'm with too much work, there could be two reasons, one is the certificate, do you have the app already installed on the TV but an earlier version? If so, try to uninstall it and try again because it detects that you have the appid installed but with a different certificate and it cannot replace the app, the second option is incompatibility of the tizen studio with the vbersion of tizen on your tv... if not Can you install it, send us the version of your TV and the version of Tizen Studio that you have installed, thanks

**Reply by Borsoi** [reply]
Hello @jhassan8 I've tried creating a tizen and a samsung certificate but got the same error as I tried to execute the installer.

About the earlier version of the app, I don't have it installed, it is the first time I'm trying to install it and I'm only trying the online version, never tried the offline one.

My Samsung TV version is: 1260.1
Tizen Studio Version: Latest - 5.1

Should I change the studio version? Can you give me some more info in how to do it?

Btw, no problem about the delay, your project is awesome and I hope I could use it.

**Comment by Juanimurano** [comment]
Hi, 

I also have the same problem, apparently it cames from the version of tizenOS. I tried to install in two diferents Samsungs TV:
TV1: TizenOS v1260.1 (with the 118 error)
TV2: TizenOS v1310.0 (after some trys it install OK)

Let me help you with any debbug or log that can give to you or any information. 

thank you

**Reply by heymrlen** [reply]
Got the same error 118 here. I hope there's a fix for this! Too bad my TV is running Tizen 2.4

**Reply by jhassan8** [reply]
Hi @heymrlen, did you try to install it from tizen studio using the device manager?

**Reply by heymrlen** [reply]
> Hi @heymrlen, did you try to install it from tizen studio using the device manager?

I did. It worked on my Samsung M5 Monitor but not on my old Samsung TV running Tizen 2.4. Got the same error of 118
